shongo-controller/src/main/java/cz/cesnet/shongo/controller/booking/request/AbstractReservationRequestAuxData.java

```diff
@@ -0,0 +1,30 @@
+package cz.cesnet.shongo.controller.booking.request;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import cz.cesnet.shongo.controller.booking.specification.Specification;
+import lombok.Getter;
+import lombok.Setter;
+import org.hibernate.annotations.Immutable;
+import org.hibernate.annotations.Type;
+
+import javax.persistence.*;
+
+@Getter
+@Setter
+@Entity
+@Immutable
+@Table(name = "arr_aux_data")
```

Comment:
Tu vytváraš tabuľku v DB s názvom arr_aux_data, ale zároveň v init.sql vytváraš view s rovnakým názvom. Ten view je zdá sa mi už len zabudnutý pozostatok na zmazanie?

Reply:
`arr_aux_data` ma skutocne byt view (ktory jednoducho mapuje `aux_data` na jednotlive fieldy, aby som to nemusel robit v jave). V `init.sql` ho definujem a entita je potom vytvorena na pracu s nim.

Reply:
Nejak nikde nevidím kde sa ten view používa. A mal by sa volať nejak inak ako tá entita. 
Snažím sa rozbehnúť projekt lokálne a dostávam potom error pri inicializácii že entita už existuje:
`Exception in thread "main" javax.persistence.PersistenceException: org.hibernate.exception.SQLGrammarException: could not execute statement
	at org.hibernate.internal.ExceptionConverterImpl.convert(ExceptionConverterImpl.java:154)
	at org.hibernate.internal.ExceptionConverterImpl.convert(ExceptionConverterImpl.java:181)
	at org.hibernate.query.internal.AbstractProducedQuery.executeUpdate(AbstractProducedQuery.java:1614)
	at cz.cesnet.shongo.controller.util.NativeQuery.executeNativeUpdate(NativeQuery.java:275)
	at cz.cesnet.shongo.controller.Controller.initializeDatabase(Controller.java:945)
	at cz.cesnet.shongo.controller.Controller.main(Controller.java:1081)
Caused by: org.hibernate.exception.SQLGrammarException: could not execute statement
	at org.hibernate.exception.internal.SQLStateConversionDelegate.convert(SQLStateConversionDelegate.java:103)
	at org.hibernate.exception.internal.StandardSQLExceptionConverter.convert(StandardSQLExceptionConverter.java:42)
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:113)
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:99)
	at org.hibernate.engine.jdbc.internal.ResultSetReturnImpl.executeUpdate(ResultSetReturnImpl.java:200)
	at org.hibernate.engine.query.spi.NativeSQLQueryPlan.performExecuteUpdate(NativeSQLQueryPlan.java:107)
	at org.hibernate.internal.SessionImpl.executeNativeUpdate(SessionImpl.java:1491)
	at org.hibernate.query.internal.NativeQueryImpl.doExecuteUpdate(NativeQueryImpl.java:295)
	at org.hibernate.query.internal.AbstractProducedQuery.executeUpdate(AbstractProducedQuery.java:1605)
	... 3 more
Caused by: org.postgresql.util.PSQLException: ERROR: "arr_aux_data" is not a view`

Reply:
Pouziva sa v `ReservationRequestManager` ako nahrada `AuxDataService`, ktora sa ti predtym nepozdavala:
```java
    private List<AuxDataMerged> getAuxData(AbstractReservationRequest reservationRequest, AuxDataFilter filter)
    {
        String queryString = "SELECT arr.tagName, rt.tag.type, arr.enabled, arr.data, rt.tag.data" +
                " FROM AbstractReservationRequestAuxData arr" +
                " JOIN ResourceSpecification res_spec ON res_spec.id = arr.specification.id" +
                " JOIN ResourceTag rt ON rt.resource.id = res_spec.resource.id" +
                " WHERE rt.tag.name = arr.tagName" +
                " AND arr.id = :id";
        ....
    }
```
Ten error vyskakuje preto, ze najprv sa vytvori table kvoli JPA az potom sa zavola `init.sql` kde sa vytvara ten view.
Pridaval som to kvoli tomu aj do `migration`, ale zabudol som to pushnut, moja chyba.